core/templates/dev/head/css/oppia.css

```diff
@@ -2162,9 +2164,8 @@ md-card.oppia-dashboard-tile {
 
 .oppia-dashboard-list-view-item {
   background: #fff;
-  margin: 20px auto 0;
-  padding: 10px 20px;
-  width: 100%;
+  margin: 20px 7.5px 0px 7.5px;
+  padding: 10px 12.5px;
```

Comment:
I wanted to give the contents on each line the same amount of width which they had previously. Since I had reduced the margin by 7.5px on each side I increased the padding by the same amount on either side so that the content is not too cramped up.